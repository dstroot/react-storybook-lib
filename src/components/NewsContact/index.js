import React from 'react';
import image from './media/jesse-sq.jpg';

const NewsContact = () => {
  return (
    <div className="media border mb-5">
      <img
        className="align-self-center rounded-circle m-3"
        src={image}
        alt="Contact"
        height="100"
      />
      <div className="media-body mt-3">
        <address className="mr-3">
          <strong>Media Contact</strong>
          <br />
          <span className="lead">Jesse Page</span>
          <br />
          <a
            className="card-link"
            href="mailto:jpage@pacificlife.com?Subject=Media%20Inquiry"
            target="_top"
          >
            jpage@pacificlife.com
          </a>
          <br />
          <a className="card-link" href="tel:+19492194575">
            (949) 219-4575
          </a>
        </address>
      </div>
    </div>
  );
};

export default NewsContact;
