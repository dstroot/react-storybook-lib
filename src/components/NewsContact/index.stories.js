import React, { Suspense } from 'react';
import { storiesOf } from '@storybook/react';

// components
import NewsContact from '../NewsContact';

// NewsContact section
storiesOf('NewsContact', module).add('show NewsContact', () => <NewsContact />);
