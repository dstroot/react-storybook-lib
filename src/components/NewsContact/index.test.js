import React, { Suspense } from 'react';
import renderer from 'react-test-renderer';
import NewsContact from '../NewsContact';

describe('NewsContact', () => {
  it('it should render', () => {
    const component = renderer.create(<NewsContact />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
