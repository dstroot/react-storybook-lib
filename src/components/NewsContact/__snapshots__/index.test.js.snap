// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsContact it should render 1`] = `
<div
  className="media border mb-5"
>
  <img
    alt="Contact"
    className="align-self-center rounded-circle m-3"
    height="100"
    src="jesse-sq.jpg"
  />
  <div
    className="media-body mt-3"
  >
    <address
      className="mr-3"
    >
      <strong>
        Media Contact
      </strong>
      <br />
      <span
        className="lead"
      >
        Jesse Page
      </span>
      <br />
      <a
        className="card-link"
        href="mailto:jpage@pacificlife.com?Subject=Media%20Inquiry"
        target="_top"
      >
        jpage@pacificlife.com
      </a>
      <br />
      <a
        className="card-link"
        href="tel:+19492194575"
      >
        (949) 219-4575
      </a>
    </address>
  </div>
</div>
`;
